staging/src/k8s.io/apimachinery/pkg/util/sets/int.go

```diff
@@ -14,86 +14,50 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by set-gen. DO NOT EDIT.
-
 package sets
 
-import (
-	"reflect"
-	"sort"
-)
-
-// sets.Int is a set of ints, implemented via map[int]struct{} for minimal memory consumption.
+// Int is a set of ints, implemented via map[int]struct{} for minimal memory consumption.
```

Comment:
Please update these comments to deprecate & tell people what they should use instead.